purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1259,11 +1259,11 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
 
     private fun logMissingProducts(
         offerings: Offerings,
-        detailsByID: HashMap<String, ProductDetails>
+        storeProductByIDStore: HashMap<String, StoreProduct>
```

Comment:
curious about the "store" suffix here... is that a convention?

Reply:
I think this is just some byproduct of a bad rename 🗡️